deploy/gk-deploy

```diff
@@ -308,29 +330,29 @@ fi
 
 heketi-cli -s http://${heketi_service} topology load --json=${TOPOLOGY}
 heketi-cli -s http://${heketi_service} setup-openshift-heketi-storage
-kubectl create -f heketi-storage.json
+${CLI} create -f heketi-storage.json
 
 check_pods "job-name=heketi-storage-copy-job" "Completed"
 
-kubectl delete all,service,jobs,deployment,secret --selector="deploy-heketi"
+${CLI} delete all,service,jobs,deployment,secret --selector="deploy-heketi"
 
-if [ "${CLI}" == "oc" ]; then
-  oc process heketi -v \
+if [ "${CLI}" == *oc* ]; then
+  ${CLI} process heketi -v \
          HEKETI_KUBE_NAMESPACE=heketi \
          HEKETI_KUBE_INSECURE=y \
          HEEKTI_KUBE_SECRETNAME="${heketi_secret}" \
-         HEKETI_KUBE_APIHOST="${kubeapi}" | oc create -f -
+         HEKETI_KUBE_APIHOST="${kubeapi}" | ${CLI} create -f -
```

Comment:
Also this seems to be a general comment on the script, not so much a comment on the changes of this patch, right?